### PR TITLE
Update single page application

### PR DESCRIPTION
**Pull request by Movgto**
# New Features:

- Single-page application refactoring
- JavaScript code for showing just the section, specified by link click

### CODE REVIEW

File: js/navigation.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      break;
+    default:
+      break;
+  }
+}
+
+// Click event listener added for each navigation link
+links.forEach((section) => {
+  section.addEventListener('click', GoToSection);
+});
```
Comment (rotimiazeez) [inline js/navigation.js:38]: Please note that on page load, all the navigation details are displayed on a single page which should not be. Kindly fix it. 👍🏽
Screenshot from 2023-07-12 22-46-39

Comment (rotimiazeez) [reply js/navigation.js:38]: At the moment, I can add a `book` without `author` and vice-versa. Please ensure that all necessary parameters are included before a book can be added. 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Movgto & @alexansaa 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good job implementing the navigation bar 👏🏽
- Three content sections - Books list, Add book form, and Contact info are implemented 👍🏽
- Good UI 👌🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: js/navigation.js (hunk 1 above)
Comment (KDlamini) [reply js/navigation.js]: Well resolved ✔️ 

image

Validation 👍🏾 

image

**Review by KDlamini — APPROVED** [review]
### Hi team @Movgto, @alexansaa 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Professional pull request
✔️ Three pages - Books list, Add book form, and Contact info are present
✔️ A navigation bar allows for navigating the three pages

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**